test/unit/event/send-submission-batch-event.test.js

```diff
@@ -0,0 +1,136 @@
+const mockSendEvents = jest.fn()
+const mockPublishEvents = jest.fn()
+const MockPublishEventBatch = jest.fn().mockImplementation(() => {
+  return {
+    sendEvents: mockSendEvents
+  }
+})
+const MockEventPublisher = jest.fn().mockImplementation(() => {
+  return {
+    publishEvents: mockPublishEvents
+  }
+})
+jest.mock('ffc-pay-event-publisher', () => {
+  return {
+    PublishEventBatch: MockPublishEventBatch,
+    EventPublisher: MockEventPublisher
+  }
+})
+jest.mock('../../../app/config')
+const config = require('../../../app/config')
+const { PAYMENT_SUBMITTED } = require('../../../app/constants/events')
+const { SOURCE } = require('../../../app/constants/source')
+const sendSubmissionEvents = require('../../../app/event/send-submission-batch-event')
+
+let batch
+let paymentRequest
+let filename
+
+beforeEach(() => {
+  paymentRequest = JSON.parse(JSON.stringify(require('../../mocks/payment-request')))
+  batch = JSON.parse(JSON.stringify(require('../../mocks/batch')))
+  batch.paymentRequests = [paymentRequest, paymentRequest]
+
+  filename = 'test.csv'
```

Comment:
Similar comment to this: https://github.com/DEFRA/ffc-pay-responses/pull/28#discussion_r1133789997

Reply:
Updated